components/next_steps_view/next_steps_view.scss

```diff
@@ -0,0 +1,88 @@
+#app-content.NextStepsView {
+    background-image: url(../../images/onboarding-bg.svg);
+    background-repeat: no-repeat;
+    background-position-y: bottom;
+    background-position-x: -305px;
+    background-size: calc(100% + 305px);
+}
+
+.NextStepsView__header {
+    padding: 32px 40px;
+    display: flex;
+}
+
+.NextStepsView__header-headerText {
+
+}
+
+.NextStepsView__header-headerTopText {
+    font-weight: 600;
+    font-size: 32px;
+    line-height: 40px;
+    letter-spacing: -0.02em;
+}
+
+.NextStepsView__header-headerBottomText {
+    font-size: 16px;
+    line-height: 24px;
+    margin-top: 8px;
+}
+
+.NextStepsView__header-logo {
+    margin-left: auto;
```

Comment:
0/5: Maybe stick with flexbox to push the logo to the edge since you're already using it? Either add `flex-grow: 1` to`.NextStepsView__header-headerText` or `justify-content: space-between` to `.NextStepsView__header`.

Reply:
This was what I was told to use to right-align one flexbox item to the right. But `flex-grow: 1` works equally as well so I'll use that.